### ISSUE TITLE
More descriptive error voting for >10 val groups

### ISSUE THREAD

**Issue by jmrossy** [celo-tools/celo-web-wallet#41]
#### Feature Description

The limit is 10: https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/governance/Election.sol#L260

The current error is unclear (generic tx error)

**Comment by RedGodfather** [comment]
So by more descriptive you mean concatenate the number after the msg? Something like: "Voted for too many groups, please limit number of groups to 10(let's say)"?

**Comment by jmrossy** [comment]
> So by more descriptive you mean concatenate the number after the msg, the limit being maxNumGroupsVotedFor? Something like: "Voted for too many groups, please limit number of groups to 10(let's say)"?

Yeah something like that. We should add a condition in the validate method to catch this even before the user attempts the tx: https://github.com/celo-tools/celo-web-wallet/blob/master/src/features/validators/stakeToken.ts#L39
